common/src/python/curator/scheduling.py

```diff
@@ -334,9 +318,7 @@ def __compute_cores(self) -> int:
     def apply(self,
               context: GearToolkitContext,
               curator_type: Type[C],
-              deriver: AttributeDeriver,
-              curation_tag: str,
-              force_curate: bool = False) -> None:
+              curator_type_args: Optional[Dict[str, Any]] = None) -> None:
```

Comment:
I'd rather have explicit arguments so that typing can help identify errors when there are some.